_posts/2021-06-24-debezium-1-6-cr1-released.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  Debezium 1.6.0.CR1 Released
+date:   2021-06-24
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, outbox ]
+author: ccranfor
+---
+
+It's my pleasure to announce the release of Debezium *1.6.0.CR1*!
+
+This release adds skipped operations optimizations for SQL Server, introduces Heartbeat support to the Oracle connector,
+Oracle BLOB/CLOB support is now opt-in only, and provides a range of bug fixes and other improvements across different Debezium connectors.
+
++++<!-- more -->+++
+
+== Skipped operations optimizations - SQL Server
+
+Debezium introduced the notion of `skipped.operations` some time ago.
+This feature was to allow connectors to control what change events were emitted during the streaming phase, optionally omitting certain changes such as updates or deletes.
+
+The `skipped.operations` feature was originally implemented to act more like an SMT where it was a filter that was applied toward the end of the CDC pipeline just before an event was emitted.
+While this worked well, there was an optimization that could be achieved here because SQL Server records the change type in the capture table along side the change data.
+Using this knowledge, the query to fetch changes from the capture table was adjusted so that the changes returned from SQL Server are pre-filtered based on these "skipped operations".
+
+Overall, this optimization should improve both the time spent during iterations of capturing changes when the conenctor is configured to use `skipped.operations` as well as reduce the overall garbage collection that the connector may do over its lifetime due to generating fewer objects that are later discarded.
+
+Many thanks to https://github.com/blcksrx[Hossein Torabi] for identifying this optimization and contributing it!
+
+== Oracle Heartbeat Support
+
+Debezium supports heartbeats across its other connectors, this feature is now available to Oracle!
+
+A heartbeat is a critical part of insuring that a connector's offsets remain synchronized with Kafka.
+Normally, the offsets are automatically maintained by Kafka when we emit an event to a topic.
+But there are circumstances that occur in environments where event emission can be infrequent enough that stale offsets become a concern.
+
+Regardless which adapter the Oracle connector is deployed to use, there are situations where events may be seen by the connector or offset state is updated but because we don't emit an event to Kafka, the offsets maintained locally by the connector may become stale.
```

Comment:
```suggestion
Regardless which adapter the Debezium Oracle connector is deployed to use, there are situations where events may be seen by the connector or offset state is updated but because we don't emit an event to Kafka, the offsets maintained locally by the connector may become stale.
```